docs/user_guide/docker.rst

```diff
@@ -1,11 +1,45 @@
 
-How to run traffic in a Docker container?
+How to use traffic in a Docker container?
 =========================================
 
-Run a Jupyter environment with traffic (latest)
------------------------------------------------
+If you don't want to be bothered with python's dependencies and just want to work, using a docker container might be an easy solution. It is quite simple to run the traffic library with everything it needs in a Docker container and you have a few options. We provide a self contained container image on GitHub that you can use. It has everything in it that you need. Or, alternatively, you can build your own image. We describe both below.
 
-It is quite simple to run the traffic library in a Docker container. The
+Use the GitHub container image
+------------------------------
+
+Using the container image from GitHub is simple. Just make sure you have docker installed, then pull the image with
+
+.. code:: bash
+
+    docker pull ghcr.io/xoolive/traffic/jupyter:latest
+
+Once the image is downloaded, you can run it with
+
+. code:: bash
+
+    docker run -it -p 8888:8888 ghcr.io/xoolive/traffic/jupyter
+
+This will start the container and you can connect to it with the browser and work in a jupyter notebook. Just copy-paste the URL that starts with *http://127.0.0.1:8888...* into your favorite browser and have fun. 
+
+Use the GitHub container image with Visual Studio Code
+------------------------------------------------------
+
+If you use VS Code, life is even more convenient. You just need to make sure that you have the *Docker* and *Remote - Containers* extensions installed.
```

Comment:
Let's just add those extensions to https://github.com/xoolive/traffic/blob/master/.vscode/extensions.json (but not change anything to the text)

Reply:
I added them.